diff --git a/run_tests.sh b/run_tests.sh
index 48ce6583b..5fffe12b6 100755
--- a/run_tests.sh
+++ b/run_tests.sh
@@ -54,7 +54,7 @@ source /etc/os-release || source /usr/lib/os-release
 # Figure out the appropriate package install command
 case ${ID,,} in
     *suse*) pkg_mgr_cmd="zypper -n in" ;;
-    centos|rhel|fedora) pkg_mgr_cmd="dnf install -y" ;;
+    rocky|rhel|fedora) pkg_mgr_cmd="dnf install -y" ;;
     ubuntu|debian) pkg_mgr_cmd="apt-get install -y" ;;
     # Gentoo needs to have version set since it's rolling
     gentoo) pkg_mgr_cmd="emerge --jobs=4"; VERSION="rolling" ;;
diff --git a/scripts/bootstrap-ansible.sh b/scripts/bootstrap-ansible.sh
index 49a122a00..c464ffe72 100755
--- a/scripts/bootstrap-ansible.sh
+++ b/scripts/bootstrap-ansible.sh
@@ -71,7 +71,7 @@ determine_distro

 # Install the base packages
 case ${DISTRO_ID} in
-    centos|rhel)
+    rocky|rhel)
         dnf -y install \
           git curl autoconf gcc gcc-c++ nc \
           python3 python3-devel libselinux-python3 \
diff --git a/scripts/scripts-library.sh b/scripts/scripts-library.sh
index 742366cde..8d5ec1db2 100755
--- a/scripts/scripts-library.sh
+++ b/scripts/scripts-library.sh
@@ -212,7 +212,7 @@ function setup_ara {
 function run_dstat {
   if [ "$GATE_EXIT_RUN_DSTAT" == true ]; then
     case ${DISTRO_ID} in
-      centos|rhel)
+      rocky|rhel)
           dnf -y install dstat
           ;;
       ubuntu)
@@ -261,7 +261,7 @@ function log_instance_info {
           apt-get update
           DEBIAN_FRONTEND=noninteractive apt-get -y install iproute2 net-tools
           ;;
-      centos|rhel)
+      rocky|rhel)
           dnf -y install iproute
           ;;
   esac
@@ -336,7 +336,7 @@ function get_instance_info {

   determine_distro
   case ${DISTRO_ID} in
-      centos|rhel|fedora|opensuse)
+      rocky|rhel|fedora|opensuse)
           rpm -qa | sort > \
             "/openstack/log/instance-info/host_packages_info_${TS}.log" || true
           ;;
